pytensor/tensor/extra_ops.py

```diff
@@ -592,6 +593,13 @@ def squeeze(x, axis=None):
         # Nothing to do
         return _x
 
+    axes = ()
```

Comment:
Confusing name, now we have `axes` and `axis`. We should call this intermediate variable something more obvious. Also all this can be done in a list comprehension, which is more pythonic/readbale:

Reply:
Also still missing the comment that explains what is happening/why this is needed